core-bundle/src/Command/MigrateCommand.php

```diff
@@ -519,10 +519,10 @@ private function compileConfigurationErrors(): array
                 return $errors;
             }
 
-            $row = $this->connection->fetchAssociative("SHOW VARIABLES LIKE 'innodb_large_prefix'");
+            $largePrefixSetting = $this->connection->fetchAssociative("SHOW VARIABLES LIKE 'innodb_large_prefix'");
```

Comment:
```suggestion
            $largePrefixSetting = $this->connection->fetchAssociative("SHOW VARIABLES LIKE 'innodb_large_prefix'")['Value'] ?? '';
```

Reply:
Just transferred this code "as is" from the install tool, but it makes perfectly sense to adjust. Especially now that there are tests. :+1: